Sources/Qit/Extensions/SwiftUI/View+ConditionalModifier.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  View+ConditionalModifier.swift
+//  Qit
+//
+//  Created by Alexandre Nussbaumer on 09.06.22.
+//  Copyright © 2022 QoQa Services SA. All rights reserved.
+//
+
+import SwiftUI
+
+extension View {
+
+    typealias ContentTransform<Content: View> = (Self) -> Content
+
+    /// Conditional `SwiftUI` view modifier
+    /// - Parameters:
+    ///   - condition: VIew modifier condition
```

Comment:
I made a small typo here: 
```suggestion
    ///   - condition: View modifier condition
```